samples/kubernetes/end2end/docs/08-alert.md

```diff
@@ -0,0 +1,81 @@
+## Firing Alerts
+In this step we show you how to fire alerts.
+
+### Configuring Alerts
+We do all the alerting related configurations in the customized [values.yaml](../prometheus/values.yaml) when installing the Prometheus chart.  
+
+First, configure alerting rules. For demonstration purpse, We define one [alerting rule](../prometheus/values.yaml#L45) named 'ClusterWarning' that will fire alerts if some WLS cluster has only one running server.
+
+Second, configure Alertmanager. In [alertmanager.yml](../prometheus/values.yaml#L27) we define one route and one receiver that will send all the received alerts to a webhook.
+
+### Verifying Configuration
+Access the Prometheus alerts page in your browser with the URL `http://<HostIP>:30000/alerts`. Check the configured alert rules. You should get something like below.  
+
+![Alert Rule](./images/alert-rule.png)  
+
+Go to Alert Manager web UI via the URL `http://<HostIP>:32000`. 
+- Check the configured rule and receiver via the URL `http://<HostIP>:32000/#/status`. You should get something similar as below.  
+
+![Alert Manager Configuration](./images/alert-manager-config.png)  
+- Check the alert page via the URL `http://<HostIP>:32000/#/alerts`. There is no received alert yet.
+
+### Triggering Alerts
+To trigger alerts, one WLS cluster need to have only one managed server running. 
+- Update file `demo-domains/domain1.yaml` to change the line from `replicas: 2` to `replicas: 1`.
+- Apply the updated domain resource file.
+  ```
+  kubectl apply -f demo-domains/domain1.yaml
+  ```
+If the new domain resource is applied successfully, one WLS server will start to termiate. Wait until only one managed server is running.
+```
+kubectl get pod -l weblogic.domainName=domain1
+```
+> output
+```
+NAME                       READY   STATUS    RESTARTS   AGE
+domain1-admin-server       1/1     Running   1         2h
+domain1-managed-server-1   1/1     Running   1         2h
+```
+
+Then keep refreshing the Prometheus alerts page `http://<HostIP>:30000/alerts`. There will be one alert generated. First the alert is in `PENDING` state. After about 1 minutes (as configured),  the alert state is changed to `FIRING` and this is the time that the alert is actually sent to the Alert Manager. 
```

Comment:
The state of the alert at first is `PENDING` and after about 1 minute (or after configured minutes),  the state of the alert changes to  `FIRING`, it is at this time that the alert is sent to the Alert Manager.

Reply:
Could you replace "There will be one alert generated. First the alert is in `PENDING` state. After about 1 minutes (as configured),  the alert state is changed to `FIRING` and this is the time that the alert is actually sent to the Alert Manager. " for "The state of the alert at first is PENDING and after about 1 minute (or after configured minutes), the state of the alert changes to FIRING, it is at this time that the alert is sent to the Alert Manager."